src/components/v5/common/ActionSidebar/partials/MultiSigSidebar/partials/MultiSigWidget/partials/FinalizeStep/FinalizeStep.tsx

```diff
@@ -179,8 +183,9 @@ const FinalizeStep: FC<FinalizeStepProps> = ({
   useEffect(() => {
     if (isMultiSigExecuted || isMultiSigRejected) {
       setIsFinalizePending(false);
+      handleMultiSigFinalized(action);
```

Comment:
I think we want to call this only if it's executed, so we don't update the member status if it was rejected